target/linux/meson/files/arch/arm/boot/dts/meson8b-onecloud.dts

```diff
@@ -0,0 +1,389 @@
+// SPDX-License-Identifier: GPL-2.0 OR MIT
+
+/dts-v1/;
+#include "meson8b.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "Thunder OneCloud";
+	compatible = "thunder,onecloud", "amlogic,meson8b";
+
+	aliases {
+		serial0 = &uart_AO;
+		mmc0 = &sd_card_slot;
+		mmc1 = &sdhc;
+	};
+
+	chosen {
+		stdout-path = "serial0:115200n8";
+	};
+
+	memory {
+		device_type = "memory";
+		reg = <0x40000000 0x40000000>;
+	};
+
+	emmc_pwrseq: emmc-pwrseq {
+		compatible = "mmc-pwrseq-emmc";
+		reset-gpios = <&gpio BOOT_9 GPIO_ACTIVE_LOW>;
+	};
+
+	gpio-keys {
+		// compatible = "gpio-keys-polled";
+		// poll-interval = <100>;
+
+		compatible = "gpio-keys";
+
+		autorepeat;
+
+		reset-button {
+			label = "reset";
+			linux,code = <BTN_0>;
+
+			// gpios = <&gpio_ao GPIOAO_5 GPIO_ACTIVE_LOW>;
+
+			interrupt-parent = <&gpio_intc>;
+			interrupts = <5 IRQ_TYPE_LEVEL_LOW>; // GPIOAO 5
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_status_red: status-red {
+			label = "red:status";
+			gpios = <&gpio_ao GPIOAO_2 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_status_green: status-green {
+			label = "green:status";
+			gpios = <&gpio_ao GPIOAO_3 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_status_blue: status-blue {
+			label = "blue:status";
+			gpios = <&gpio_ao GPIOAO_4 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	p12v: regulator-p12v {
```

Comment:
it should be named as `<name>-regulator`, e.g. `p12v-regulator`, the same applies below